domain.go

```diff
@@ -1710,7 +1710,7 @@ func (d *Domain) MemoryStats(nrStats uint32, flags uint32) ([]DomainMemoryStat,
 		return []DomainMemoryStat{}, GetLastError()
 	}
 
-	out := make([]DomainMemoryStat, result)
+	out := make([]DomainMemoryStat, 0, result)
```

Comment:
I'm removing the ', result' parameter, since in other cases in this file we don't bother to specify an explicit capacity upfront.